Kernel/Random.h

```diff
@@ -182,7 +194,7 @@ class EntropySource {
 //       The only difference is that get_fast_random is guaranteed not to block.
 
 void get_fast_random_bytes(u8*, size_t);
-void get_good_random_bytes(u8*, size_t);
+bool get_good_random_bytes(u8*, size_t, bool = true, bool = true);
```

Comment:
Please give names to these arguments as it's super non-obvious what they are (and they are the same type, too!)